coops-ui/src/components/AnnualReport/BCorp/ARDate.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <v-card flat class="container">
+    <div class="timeline">
+        <label>Annual Report Date</label>
+        <span class="date ar-date">{{this.toReadableDate(this.nextARDate)}}</span>
+        <label>Filing Date</label>
+        <span class="date file-date">Today ({{this.toReadableDate(new Date())}})</span>
+      </div>
+  </v-card>
+</template>w
+<script lang="ts">
+import { Component } from 'vue-property-decorator'
+import { mapState } from 'vuex'
+import DateMixin from '@/mixins/date-mixin'
+
+@Component({
+  computed: {
+    ...mapState(['nextARDate'])
+  }
+})
+export default class ARDate extends DateMixin {
+    readonly nextARDate: string
```

Comment:
Severin: You probably need to use the non-null assertion operator here so you don't get an error in the template when you use 'nextARDate' before it's assigned here. See AnnualReport/AGMDate.vue for an example of this.

Cameron: Easily done, but not sure where this error is occurring in this instance with regards to ARDate

Severin: I can see in the template that you use `this.nextARDate`. I'm surprised you're not getting an error (lint or runtime) about using that while it's technically undefined.

Reply:
I see this is fixed now.